src/components/connections/Connection.ts

```diff
@@ -1,17 +1,18 @@
 import * as vscode from "vscode";
-import { ViewEngine as QueryViewEngine, APIRequest, Result } from "../Query";
+import { ViewEngine as QueryViewEngine, APIRequest } from "../Query";
 import { INode } from "./INode";
 import { Status } from "./Status";
 import { ConnectionNode, InfluxDBConectionsKey } from "./ConnectionNode";
 import { EditConnectionView } from "./EditConnectionView";
 import { ExtensionContext } from "vscode";
+import { TableResult } from "../TableView";
 
 const uuidv1 = require("uuid/v1");
 export interface InfluxDBConnection {
   readonly version: InfluxConnectionVersion;
   readonly id: string;
   readonly name: string;
-  readonly hostNport: string;
+  readonly host: string;
```

Comment:
I see, this is the one breaks the existing connection

Reply:
is that because it gets it out of vscode's store?

Reply:
yeah, it pulled from global state, which is like a json string of existing connections.